models/staging/base_parta_diagnosis_code.sql

```diff
@@ -0,0 +1,53 @@
+with diagnosis as (
+
+    select
+          {{ cast_string_or_varchar('cur_clm_uniq_id') }} as cur_clm_uniq_id
+        , {{ cast_string_or_varchar('bene_mbi_id') }} as bene_mbi_id
+        , {{ cast_string_or_varchar('bene_hic_num') }} as bene_hic_num
+        , {{ cast_string_or_varchar('clm_type_cd') }} as clm_type_cd
+        , {{ cast_string_or_varchar('clm_prod_type_cd') }} as clm_prod_type_cd
+        , {{ cast_string_or_varchar('clm_val_sqnc_num') }} as clm_val_sqnc_num
+        , {{ cast_string_or_varchar('clm_dgns_cd') }} as clm_dgns_cd
+        , {{ cast_string_or_varchar('bene_eqtbl_bic_hicn_num') }} as bene_eqtbl_bic_hicn_num
+        , {{ cast_string_or_varchar('prvdr_oscar_num') }} as prvdr_oscar_num
+        , {{ cast_string_or_varchar('clm_from_dt') }} as clm_from_dt
+        , {{ cast_string_or_varchar('clm_thru_dt') }} as clm_thru_dt
+        , {{ cast_string_or_varchar('clm_poa_ind') }} as clm_poa_ind
+        , {{ cast_string_or_varchar('dgns_prcdr_icd_ind') }} as dgns_prcdr_icd_ind
+    from {{ var('parta_diagnosis_code') }}
+
+),
+
+diagnosis_seed as (
```

Comment:
This is a hacky workaround. I would love feedback on how to do this better. This is how I solved the problem of missing diagnosis and procedures. This seed is included in the pivot to ensure we have 1-25 columns for dx/px.

Reply:
@sarah-tuva - your solution here is good.  It lines up with my initial thoughts on how to solve the problem.